src/main/java/org/folio/circulation/domain/Account.java

```diff
@@ -29,15 +47,23 @@ public String getStatus() {
     return getNestedStringProperty(representation, "status", "name");
   }
 
+  public Optional<DateTime> getClosedDate() {
+
+    return feeFineActions.stream()
```

Comment:
It can't be null, it will be an empty list. When a fee/fine is created via Folio, it will contait at least one fee fine action. When at least one feefine action has zero balance, it means the fee/fine has been paid - it is closed.

Reply:
If it should never be null, please can an appropriate exception be thrown with a meaningful message when the constructor parameter is null

Reply:
Added a null-check